Directory.Build.props

```diff
@@ -3,24 +3,59 @@
     Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), AspNetCoreSettings.props))\AspNetCoreSettings.props"
     Condition=" '$(CI)' != 'true' AND '$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), AspNetCoreSettings.props))' != '' " />
 
-  <Import Project="version.props" />
+  <PropertyGroup>
+    <UsingToolXliff>false</UsingToolXliff>
+  </PropertyGroup>
+
+  <!--
+    We don't follow their conventions for project naming.
+  -->
+  <PropertyGroup Condition="'$(IsUnitTestProject)' == ''">
+    <IsUnitTestProject>false</IsUnitTestProject>
+    <IsUnitTestProject Condition="$(MSBuildProjectName.EndsWith('.Tests'))">true</IsUnitTestProject>
+    <IsUnitTestProject Condition="$(MSBuildProjectName.EndsWith('.FunctionalTests'))">true</IsUnitTestProject>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(IsUnitTestProject)' == 'false'">
+    <IsPackable>true</IsPackable>
+    <IsShipping>true</IsShipping>
+  </PropertyGroup>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
 
   <PropertyGroup>
+    <Authors Condition="'$(Authors)' == ''">Microsoft</Authors>
```

Comment:
These duplicate what is already defined by Arcade:  https://github.com/dotnet/arcade/blob/c9f5d6ad24d3078f004168fbd4c476d489a87510/src/Microsoft.DotNet.Arcade.Sdk/tools/ProjectDefaults.props#L12. You can remove the following:

* Authors
* Copyright
* Serviceable
* NeutralLanguage 
* PackageRequireLicenseAcceptance
* SuppressNETCoreSdkPreviewMessage